docs/references/debugging-sdk-tools.md

```diff
@@ -0,0 +1,82 @@
+# Debugging SDK Tools
+Now that you've built your first SDK Tool, you want to confirm that it's properly installed and that the code you've written is being executed. This guide will show you places to check for correctness and how to verify that your plugin is being correctly built and executed. It is recommended to follow these steps when behavior is not acting as expected, and to always rebuild using the `create-yxi` command to iterate and ensure that changes are being made to your plugin code.  
+
+
+## Table of Contents
+1. After `create-yxi`
+2. After installing `.yxi`
+3. After running in Designer
+4. Running the plugin at Command Line
+5. Simple ways to print/check values
```

Comment:
TW review:

1. After `create-yxi`
2. After Installing the YXI
3. After Running in Designer
4. Running the Plugin via Command Line
5. Simple Ways to Print and Check Values